***Started - reading and parsing pddl file ...***
***Finished reading and parsing pddl file after 0.113 seconds CPU time, 0.113 seconds wall-clock time***

***Started - preprocessing tarski problem ...***
***Finished preprocessing tarski problem after 0.000 seconds CPU time, 0.000 seconds wall-clock time***

***Started - grounding ...***
/tmp/tmpg50aa0g9:6:10-11: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:7:27-28: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:9:39-40: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:10:27-28: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:12:38-39: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:13:47-48: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:15:40-41: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:17:43-44: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:19:27-28: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:20:33-34: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:22:32-33: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:24:32-33: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:26:32-33: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:28:37-38: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:29:27-28: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:31:37-38: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:32:46-47: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:34:39-40: error: syntax error, unexpected (

/tmp/tmpg50aa0g9:36:42-43: error: syntax error, unexpected (

*** ERROR: (gringo): grounding stopped because of errors
Traceback (most recent call last):
  File "brfs.py", line 60, in <module>
    main( sys.argv[1], sys.argv[2], sys.argv[3] )
  File "brfs.py", line 15, in main
    ground_generate_task( domain_file, problem_file, task )
  File "/home/anubhav-cs/Documents/Research_Project/anubhav-research/LAPKT-public/planners/brfs-tarski/tarski_parser/tarski_util.py", line 185, in ground_generate_task
    reachable_action_params =   copy.deepcopy( grounding.ground_actions())
  File "/media/anubhav-cs/SSD256/Applications/tarski/src/tarski/grounding/lp_grounding.py", line 39, in ground_actions
    model = self._solve_lp()
  File "/media/anubhav-cs/SSD256/Applications/tarski/src/tarski/grounding/lp_grounding.py", line 50, in _solve_lp
    model_filename = run_clingo(lp)
  File "/media/anubhav-cs/SSD256/Applications/tarski/src/tarski/reachability/clingo_wrapper.py", line 26, in run_clingo
    raise ExternalCommandError("Gringo exited with code {}".format(retcode))
tarski.errors.ExternalCommandError: Gringo exited with code 1
